staging/src/k8s.io/cloud-provider/controllers/node/node_controller.go

```diff
@@ -155,7 +155,12 @@ func (cnc *CloudNodeController) UpdateNodeStatus(ctx context.Context) {
 	}
 
 	for i := range nodes.Items {
-		cnc.updateNodeAddress(ctx, &nodes.Items[i])
+		instanceMetadata, err := cnc.getInstanceNodeAddresses(ctx, &nodes.Items[i])
+		if err != nil {
+			klog.Errorf("Error get instance metadata: %v", err)
```

Comment:
nit: "Error getting instance metadata for node addresses: %v"

Reply:
done